taichi/ir/frontend_ir.h

```diff
@@ -730,7 +730,7 @@ class SNodeOpExpression : public Expression {
   SNode *snode;
   SNodeOpType op_type;
   ExprGroup indices;
-  Expr value;
+  std::vector<Expr> values;
```

Comment:
Can we add a comment here stating that `values` are only used for `op_type == SNodeOpType::append`